src/TrixiParticles.jl

```diff
@@ -6,8 +6,10 @@ using Adapt: Adapt
 using CSV: CSV
 using Dates
 using DataFrames: DataFrame
+using DelimitedFiles
 using DiffEqCallbacks: PeriodicCallback, PeriodicCallbackAffect, PresetTimeCallback
 using FastPow: @fastpow
+using FileIO: FileIO, File, query, @format_str, Stream, stream
```

Comment:
1. I would not import `query`, and instead use it as `FileIO.query`, as not to confuse users where `query` is defined.
2. `@format_str` is not used, is it?
3. Since `File`, `Stream` and `stream` are only used once or twice, you might want to think about calling these explicitly as `FileIO.` as well to make things clearer.